# reason: https://github.com/proot-me/proot/issues/106
# src: Debian10

From: Simon McVittie <smcv@debian.org>
Date: Sat, 14 Jan 2017 15:52:13 +0000
X-Dgit-Generated: 5.1.0-1.2 dcc5f0999759be03aed5b62a8683e0b965d0219d
Subject: Disable seccomp-based tracing performance improvement

It is faster (according to upstream documentation), but on current
(4.8.4+) kernels it just segfaults. Software that works slowly seems
better than software that doesn't work at all.

Bug: https://github.com/proot-me/PRoot/issues/106

---

--- a/src/GNUmakefile
+++ b/src/GNUmakefile
@@ -105,7 +105,7 @@ CHECK_VERSION = VERSION=$$($(GIT) descri
 		then /bin/echo -e "\#undef VERSION\n\#define VERSION \"$${VERSION}\""; \
 		fi;
 
-CHECK_FEATURES = process_vm seccomp_filter
+CHECK_FEATURES = process_vm
 CHECK_PROGRAMS = $(foreach feature,$(CHECK_FEATURES),.check_$(feature))
 CHECK_OBJECTS  = $(foreach feature,$(CHECK_FEATURES),.check_$(feature).o)
 CHECK_RESULTS  = $(foreach feature,$(CHECK_FEATURES),.check_$(feature).res)

